src/components/EventOverview/UserProfile.tsx

```diff
@@ -35,6 +35,7 @@ const UserProfile = () => {
     // TODO: Redirect to edit profile
   };
   const handleLogoutClick = (event) => {
+    fetch("http://localhost:4000/logout");
```

Comment:
this will eventually logout the current user, but i'm running into a CORS issue right now

Reply:
I removed this line in the latest commit to prevent errors - this will eventually be added back in when I tackle logout functionality